packages/suite/src/utils/wallet/accountUtils.ts

```diff
@@ -462,15 +462,15 @@ export const getAccountIdentifier = (account: Account) => {
 
 export const accountSearchFn = (
     account: Account,
-    rawSearchString: string,
+    rawSearchString: string | undefined,
     coinFilter?: Account['symbol'],
 ) => {
     // if coin filter is active and account symbol doesn't match return false and don't continue the search
     const coinFilterMatch = coinFilter ? account.symbol === coinFilter : true;
     if (!coinFilterMatch) return false;
 
-    const searchString = rawSearchString.trim().toLowerCase();
-    if (searchString.length === 0) return true; // no search string
+    const searchString = rawSearchString?.trim().toLowerCase();
+    if (!searchString || searchString.length === 0) return true; // no search string
```

Comment:
`!searchString` is enough, js check returns empty string as `false` so it covers both cases

Reply:
huh, TIL!